README.md

```diff
@@ -1,14 +1,28 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
 ## Latest Release: v2.12
+
 [Read the specification](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md)
 
 [Release Notes for past versions](release-notes.md)
 
-## Working Changes
+## SDLC
+
+### Working Changes
```

Comment:
I thought about that too as I was writing my previous comment - but "RC" might imply its about to go out, and most of that time that's not true.  We need a nicer term for "Bleeding Edge Spec".